### ISSUE TITLE
NoClassDefFoundError on server start-up

### ISSUE THREAD

**Issue by SlimeDog** [PikaMug/Quests#1150]
Spigot 1.15.2
Depenizen 2.0.0-b610
Quests 3.9.0-b84

On server start-up, the following ERROR is logged. This did not occur on Quests 3.8.9-b82, with same Spigot and Depenizen versions.
```
[23:32:13] [Server thread/INFO]: ^[[0;35;1m ^[[0;31;1mERROR!
^[[0;37;22m     Error Message: ^[[0;37;1mCould not register command com.denizenscript.depenizen.bukkit.commands.quests.QuestsCommand, exception
                   follows... ^[[m
[23:32:13] [Server thread/INFO]: ^[[0;35;1m ^[[0;31;1mERROR!
^[[0;37;22m     Error Message: ^[[0;37;1mInternal exception was thrown!
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.lang.NoClassDefFoundError: me/blackvein/quests/exceptions/InvalidStageException
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.getDeclaredConstructors0(Native Method)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3137)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.getConstructor0(Class.java:3342)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.newInstance(Class.java:556)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.denizencore.scripts.commands.CommandRegistry.registerCommand(CommandRegistry.java:127)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.bridges.QuestsBridge.init(QuestsBridge.java:29)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.Depenizen.loadBridge(Depenizen.java:81)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.Depenizen.onEnable(Depenizen.java:32)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:457)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:274)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Thread.run(Thread.java:834)
^[[0;37;22m[Error Continued] ^[[0;37;1mCaused by: java.lang.ClassNotFoundException: me.blackvein.quests.exceptions.InvalidStageException
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:96)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.getDeclaredConstructors0(Native Method)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3137)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.getConstructor0(Class.java:3342)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Class.newInstance(Class.java:556)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.denizencore.scripts.commands.CommandRegistry.registerCommand(CommandRegistry.java:127)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.bridges.QuestsBridge.init(QuestsBridge.java:29)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.Depenizen.loadBridge(Depenizen.java:81)
^[[0;37;22m[Error Continued] ^[[0;37;1mcom.denizenscript.depenizen.bukkit.Depenizen.onEnable(Depenizen.java:32)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464)
^[[0;37;22m[Error Continued] ^[[0;37;1morg.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:457)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:274)
^[[0;37;22m[Error Continued] ^[[0;37;1mnet.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784)
^[[0;37;22m[Error Continued] ^[[0;37;1mjava.base/java.lang.Thread.run(Thread.java:834)
^[[m
[23:32:13] [Server thread/INFO]: ^[[0;33;1m+> [Depenizen] ^[[0;37;1mDepenizen loaded bridge for 'Quests'! ^[[m
```

**Comment by PikaMug** [comment]
The error comes from Depenizen, but it's my fault. Fix has been suggested at https://github.com/DenizenScript/Depenizen/pull/314

**Comment by SlimeDog** [comment]
Thanks. I assumed that was the case. That's why I posted here. :)

**Comment by SlimeDog** [comment]
Confirmed fixed with Depenizen 2.0.0-b611
